src/Sound/Tidal/Params.hs

```diff
@@ -385,6 +385,44 @@ ringdf = pF "ringdf"
 distort :: Pattern Double -> ControlPattern
 distort = pF "distort"
 
+-- Spectral conform
```

Comment:
it seems that not all parameters are here. At least `xsdelay` and `tsdelay` are missing?

Reply:
You're totally right. I must have fucked up and accidentally deleted xsdelay, tsdelay and freeze. Will fix

Reply:
Fixed